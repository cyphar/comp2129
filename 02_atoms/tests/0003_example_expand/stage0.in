START 2 7 5
PLACE 0 0
PLACE 2 0
PLACE 0 0
PLACE 2 0
UNDO
UNDO
DISPLAY
PLACE 0 0
PLACE 2 0
PLACE 1 0
DISPLAY
PLACE 2 0
DISPLAY
UNDO
PLACE 2 0
DISPLAY
PLACE 0 1
DISPLAY
PLACE 0 0
