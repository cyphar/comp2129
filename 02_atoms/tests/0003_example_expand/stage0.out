Game Ready
Red's Turn

Green's Turn

Red's Turn

Green's Turn

Red's Turn

Green's Turn

Red's Turn


+--------------------+
|R1|  |G1|  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
+--------------------+

Green's Turn

Red's Turn

Green's Turn


+--------------------+
|  |R2|G2|  |  |  |  |
|R1|  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
+--------------------+

Red's Turn


+--------------------+
|G1|  |G1|G1|  |  |  |
|R1|G1|G1|  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
+--------------------+

Green's Turn

Red's Turn


+--------------------+
|G1|  |G1|G1|  |  |  |
|R1|G1|G1|  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
+--------------------+

Green's Turn


+--------------------+
|G1|  |G1|G1|  |  |  |
|R2|G1|G1|  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
|  |  |  |  |  |  |  |
+--------------------+

Green Wins!
